src/main/java/net/helpscout/api/model/Conversation.java

```diff
@@ -21,7 +25,7 @@
     private boolean isDraft;
     private Long number;
     private Source source;
-    private UserRef owner;
+    private MailboxUserRef owner;
```

Comment:
Changing the type, is it going to impact any users of the client,  break serialization for anyone?